src/Map/ProspectInfo.cs

```diff
@@ -30,6 +87,96 @@ public override int GetHashCode()
                 return (X * 397) ^ Z;
             }
         }
-    }
 
-}
\ No newline at end of file
+        private void ParseMessage(string message)
+        {
+            var splits = message.Split('\n');
+            for (var i = 1; i < splits.Length - 1; i++)
+            {
+                Match match = _readingParsingRegex.Match(splits[i]);
+                if (match.Success)
+                {
+                    if (!translatedDensity.TryGetValue(match.Groups["relativeDensity"].Value, out RelativeDensity relativeDensity))
+                        relativeDensity = RelativeDensity.Zero;
+
+                    Values.Add(new OreOccurence(
+                        AllOres[match.Groups["oreName"].Value],
+                        match.Groups["pageCode"].Value,
+                        relativeDensity,
+                        double.Parse(match.Groups["absoluteDensity"].Value)
+                    ));
+                } else
+                {
+                    MatchCollection matches = _tracesParsingRegex.Matches(splits[i]);
+                    foreach (Match elem in matches)
+                        Values.Add(new OreOccurence(
+                            AllOres[elem.Groups["oreName"].Value],
+                            elem.Groups["pageCode"].Value,
+                            RelativeDensity.Miniscule,
+                            0
+                        ));
+                }
+            }
+
+            foreach (var val in Values)
+                foundOres.Add(val.Name);
+        }
+
+        public string GetMessage(bool withoutHandbook = true)
```

Comment:
what is the parameter for ? It seems to be only called with true

Reply:
I was just playing around with the handbook strings and left it in case I find a way to use it. Can be removed though.